internal/redfish/redfish.go

```diff
@@ -51,6 +52,9 @@ func New(url, user, password string, insecure bool) (*APIClient, error) {
 
 func (c *APIClient) BoardInfo() (*api.Board, error) {
 	// Query the chassis data using the session token
+	if c.Service == nil {
+		return nil, errors.New("gofish service root is not available most likely due to missing username")
```

Comment:
is there any chance to set the missing username?

Reply:
go-hal could use the superuser credentials if the given username is empty.
We can do so as soon as [this](https://github.com/metal-stack/go-hal/pull/7) PR is merged.